sqlmesh/schedulers/airflow/plugin.py

```diff
@@ -37,4 +38,5 @@ def on_load(cls, *args: t.Any, **kwargs: t.Any) -> None:
                 # the schema initialization once as a workaround.
                 logger.warning("Failed to initialize the SQLMesh State Sync: %s. Retrying...", ex)
                 time.sleep(1)
-                state_sync.migrate()
+                # TODO: Update this to include default catalog once implemented for Airflow
```

Comment:
I don't think we would ever need to update this, since the `default_catalog` is needed for this one migration. So once it's released this ship has sailed. Perhaps consider removing the TODO